system_tests/helpers/forwarderconfig.py

```diff
@@ -1,65 +1,36 @@
-import json
-from enum import Enum
-
-
-class EpicsProtocol(Enum):
-    PVA = "pva"
-    CA = "ca"
-    FAKE = "fake"
+from streaming_data_types.forwarder_config_update_rf5k import serialise_rf5k, StreamInfo
+from streaming_data_types.fbschemas.forwarder_config_update_rf5k.Protocol import (
+    Protocol,
+)
+from streaming_data_types.fbschemas.forwarder_config_update_rf5k.UpdateType import (
+    UpdateType,
+)
+from typing import List
 
 
 class ForwarderConfig:
     """
-    Class that converts the pv information to a forwarder config.
+    Class that converts the pv information to a forwarder config message payload
     """
 
     def __init__(
-        self, topic, epics_protocol: EpicsProtocol = EpicsProtocol.CA, schema="f142"
+        self, topic: str, epics_protocol: Protocol = Protocol.CA, schema: str = "f142"
```

Comment:
It's only used here, so I don't think it's really necessary, but I think we should put `"f142"` in a class somewhere to avoid typos in the future.